pkg/cmd/roachtest/tests/backup_fixtures.go

```diff
@@ -376,6 +377,18 @@ func registerBackupFixtures(r registry.Registry) {
 			timeout: 48 * time.Hour,
 			skip:    "only for fixture generation",
 		},
+		{
+			hardware: makeHardwareSpecs(hardwareSpecs{workloadNode: true}),
+			scheduledBackupSpecs: makeBackupFixtureSpecs(scheduledBackupSpecs{
+				backupSpecs: backupSpecs{
+					workload:           tpccRestore{opts: tpccRestoreOptions{warehouses: 500}},
```

Comment:
Not be merged just yet, adjusting warehouse sizes and then will add an `initWorkloadViaRestore` once an initial satisfactory fixture is created.

Reply:
since this test is skipped, its fine to merge as is and open a follow up pr with the right scale when its ready.